client/src/pages/Search/MapView/RestaurantPreview.js

```diff
@@ -1,7 +1,80 @@
+import React from "react";
+import ReviewStars from "../../../components/ReviewStars";
+import { IconButton } from "@mui/material";
+import CancelIcon from "@mui/icons-material/Cancel";
+import YelpLogo from "../../../assets/images/yelp-logo.png";
+import RedirectButton from "../../../components/RedirectButton";
+
+const classes = {
+  previewContainer: {
+    height: "160px",
+    width: "100%",
+    backgroundColor: "gray",
+    display: "flex",
+    justifyContent: "space-between",
+  },
+  imageContainer: {
+    height: "100%",
+    width: "100px",
+    marginLeft: "15px",
+    objectFit: "cover",
+  },
+  detailsContainer: {
+    marginLeft: "10px",
+  },
+  nameContainer: {
+    fontSize: "30px",
+  },
+  categoriesContainer: {
+    marginTop: "70px",
+  },
+  exitContainer: {
+    marginLeft: "100px",
+  },
+  yelpButtonContainer: {
+    marginTop: "50px",
+  },
+};
+
 function RestaurantPreview({ restaurant }) {
   return (
-    <div style={{ height: "160px", width: "100%", backgroundColor: "gray" }}>
-      {restaurant.name}
+    <div style={classes.previewContainer}>
+      <div style={{ display: "flex" }}>
+        <img
+          style={classes.imageContainer}
+          src={restaurant.image_url}
+          alt={restaurant.image_url}
+        ></img>
+
+        <div style={classes.detailsContainer}>
+          <div style={classes.nameContainer}>{restaurant.name}</div>
+
+          <div>
+            <ReviewStars rating={restaurant.rating}> </ReviewStars>
+          </div>
+
+          <div>
+            {restaurant.address.address1} {restaurant.address.state}{" "}
```

Comment:
I imagine that this section is gonna be pretty long, can we store this as a variable like `restaurantAddress`, then render it here?